ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/GetBulk.java

```diff
@@ -91,25 +100,45 @@ public GetBulkResult call() throws Exception {
             getter = new FileObjectGetter(this.outputPath);
         }
 
+        if (sync) {
+            if (this.prefix == null) {
```

Comment:
This should probably be `this.prefix == null || this.prefix.isEmpty()` or use the new Guard method `Guard.isStringNullOrEmpty(this.prefix)`